dev-packages/browser-integration-tests/suites/integrations/featureFlags/launchdarkly/test.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../utils/fixtures';
+
+import { envelopeRequestParser, waitForErrorRequest } from '../../../../utils/helpers';
+// import { FLAG_BUFFER_SIZE } from '@sentry/browser'; // TODO: not picking up export atm
+const FLAG_BUFFER_SIZE = 100;
+
+sentryTest('e2e test', async ({ getLocalTestPath, page }) => {
+  await page.route('https://dsn.ingest.sentry.io/**/*', route => {
+    return route.fulfill({
+      status: 200,
+      contentType: 'application/json',
+      body: JSON.stringify({ id: 'test-id' }),
+    });
+  });
+
+  const url = await getLocalTestPath({ testDir: __dirname, skipDsnRouteHandler: true });
+  await page.goto(url);
+
+  await page.waitForFunction(() => {
+    const ldClient = (window as any).InitializeLD();
+    for (let i = 1; i <= FLAG_BUFFER_SIZE; i++) { // TODO: import constant for buffer size
+      ldClient.variation(`feat${i}`, false);
+    }
+    ldClient.variation(`feat${FLAG_BUFFER_SIZE+1}`, true); // eviction
+    ldClient.variation('feat3', true);   // update
+    return true;
+  });
+
+  const reqPromise = waitForErrorRequest(page);
+  await page.locator('#error').click();
+  const req = await reqPromise;
+  const event = envelopeRequestParser(req);
+
+  const expectedFlags = [{ flag: 'feat2', result: false }];
+  for (let i = 4; i <= FLAG_BUFFER_SIZE; i++) {
```

Comment:
I'd probably add a test where we exceed buffer size (even though theoretically it's unit tested).

Reply:
We test that on line 26
`ldClient.variation(`feat${FLAG_BUFFER_SIZE+1}`, true); // eviction`